pyvista/utilities/reader.py

```diff
@@ -1248,6 +1249,30 @@ class Plot3DMetaReader(BaseReader):
     _class_reader = staticmethod(_vtk.lazy_vtkPlot3DMetaReader)
 
 
+class Plot3DFunctionEnum(enum.IntEnum):
```

Comment:
Should we rename this to match more closely with the reader name?  Maybe it is too long though?

Reply:
The class name would be `MultiBlockPlot3DReaderFunctionEnum`. Happy to change it if that's preferred.

Reply:
Too long I think, let's keep the one you have.